src/test/java/org/opensearch/security/ResolveAPITests.java

```diff
@@ -0,0 +1,104 @@
+package org.opensearch.security;
+
+import org.apache.http.HttpStatus;
+import org.junit.Assert;
+import org.junit.Test;
+import org.opensearch.action.admin.indices.alias.IndicesAliasesRequest;
+import org.opensearch.action.admin.indices.create.CreateIndexRequest;
+import org.opensearch.action.index.IndexRequest;
+import org.opensearch.action.support.WriteRequest;
+import org.opensearch.client.transport.TransportClient;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.xcontent.XContentType;
+import org.opensearch.security.support.ConfigConstants;
+import org.opensearch.security.test.DynamicSecurityConfig;
+import org.opensearch.security.test.SingleClusterTest;
+import org.opensearch.security.test.helper.rest.RestHelper;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+public class ResolveAPITests extends SingleClusterTest {
+
+    @Test
+    public void testResolveDnfofFalse() throws Exception {
+
+        Settings settings = Settings.builder().build();
+
+        setup(settings);
+        setupIndices();
+
+        final RestHelper rh = nonSslRestHelper();
+        RestHelper.HttpResponse res;
+
+        Assert.assertEquals(HttpStatus.SC_OK, (res = rh.executeGetRequest("_resolve/index/*?pretty", encodeBasicHeader("nagilum", "nagilum"))).getStatusCode());
```

Comment:
should test `_resolve/index/*` and `_resolve/index/<patter>*`. Applies to all other cases.

Reply:
pattern tests added